vms/platformvm/vm_regression_test.go

```diff
@@ -25,16 +27,20 @@ import (
 	"github.com/ava-labs/avalanchego/snow/validators"
 	"github.com/ava-labs/avalanchego/utils"
 	"github.com/ava-labs/avalanchego/utils/constants"
+	"github.com/ava-labs/avalanchego/utils/crypto/bls"
 	"github.com/ava-labs/avalanchego/utils/crypto/secp256k1"
 	"github.com/ava-labs/avalanchego/version"
 	"github.com/ava-labs/avalanchego/vms/components/avax"
 	"github.com/ava-labs/avalanchego/vms/platformvm/blocks"
+	blockexecutor "github.com/ava-labs/avalanchego/vms/platformvm/blocks/executor"
```

Comment:
could be wrong but I feel like we usually put aliased imports after a newline at the bottom of the imports list

Reply:
done